src/wawa-table.ts

```diff
@@ -114,7 +113,7 @@ export class WawaTable extends LitElement {
             this.fetchData(this.pageNumber, this.pageSize).then(async items => {
                 this.moreItems = items.length > 0;
                 for(let i: number = 0; i < items.length; i++) {
-                    this.items.push(new WawaItem(items[i], this));
+                    this.pushItem(items[i]);
```

Comment:
having a protected pushItem(...) method allows overloading to add functionality when an item is added